src/myuw-help.js

```diff
@@ -111,12 +131,44 @@ class MyUWHelp extends HTMLElement {
           // open the dialog
           this.setAttribute('open', '');
           this.setDialogPosition();
-          this.$dialog.focus();
+          firstTabFocus.focus();
         }
         break;
     }
   }
 
+  /**
+   * Focus trap
+   */
+  dialogFocusTrap(e) {
+    let firstTabFocus = this.focusableElements[0];
+    let lastTabFocus = this.focusableElements[this.focusableElements.length - 1];
+    let closeDialogBtn = this.$dialogCloseButton;
+
+    // Pressing Tab
+    if ((e.keyCode === 9) && this.focusableElements.length) {
```

Comment:
`keyCode` is deprecated, but you could replace it (and the comment) with `e.key = 'Tab'`

Reply:
I corrected that.